mysql/README.md

```diff
@@ -1,59 +1,143 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog Agent can collect many metrics from MySQL databases, including those for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
```

Comment:
This looks great, but I'd recommend a list format over bullet points so that the text can be reused in other parts of the product.